Mobile/ShopLens/Droid/Activities/ShoppingCartActivity.cs

```diff
@@ -11,13 +11,19 @@
 using System.Threading;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
+using Android.Support.Design.Widget;
+using Android.Support.V4.Widget;
+using SupportToolbar = Android.Support.V7.Widget.Toolbar;
+using Android.Support.V7.App;
 using Android.Views;
 using ShopLens.Droid.Activities;
+using ActionBarDrawerToggle = Android.Support.V7.App.ActionBarDrawerToggle;
 
 namespace ShopLens.Droid
 {
-    [Activity(Label = "ShoppingCartActivity", Theme = "@style/ShopLensTheme")]
-    public class ShoppingCartActivity : Activity
+    // TODO: Make voice interaction in shopping cart fluent.
+    [Activity(Label = "Shopping Cart", Theme = "@style/ShopLensTheme")]
+    public class ShoppingCartActivity : AppCompatActivity
```

Comment:
You forgot about this.
![image](https://user-images.githubusercontent.com/43152072/49827482-14c04e80-fd92-11e8-9134-746ff871b0c0.png)